catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/notes/Note.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.notes;
+
+import ddf.catalog.data.Metacard;
+
+public interface Note {
+
+    /**
+     * The metacard ID of the parent object.
+     * Uses original taxonomy to preserve backwards compatibility.
+     */
+    String PARENT_ID = Metacard.RELATED;
```

Comment:
The original taxonomy uses RELATED & DERIVED as a directional relation from parent to child. It might be best to create a different attribute for a reversed direction relation.

Reply:
But it is related, and there's nothing in the javadocs for RELATED that limit its use.

Reply:
Correct, I'm just saying we get a lot more power in the future if we keep all the relations semantically equivalent.